receiver/opencensusreceiver/config.go

```diff
@@ -99,6 +103,10 @@ func (rOpts *Config) grpcServerOptions() []grpc.ServerOption {
 	if rOpts.MaxConcurrentStreams > 0 {
 		grpcServerOptions = append(grpcServerOptions, grpc.MaxConcurrentStreams(rOpts.MaxConcurrentStreams))
 	}
+	// The default values referenced in the google docs are set by the GRPC server, so this code doesn't need
```

Comment:
What "google docs" does this refer to?

Reply:
The actual google golang docs for the GRPC server that are mentioned in the struct comments.

Reply:
It is confusing. I thought this was a reference to a particular document stored in Google Docs. Perhaps change to "GRPC docs".